Dockerfile.local

```diff
@@ -1,32 +1,12 @@
-FROM ubuntu:xenial
+FROM node:6
 MAINTAINER LiskHQ
 LABEL description="Lisk Docker Image - Local Testnet" version="1.4.2"
 
 # Install Essentials
 WORKDIR /root
 RUN apt-get update
-RUN apt-get install -y autoconf automake build-essential curl git gzip libtool nano python wget tar jq sudo
-
-# Install Node.js and tools
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
-RUN apt-get install -y nodejs
-RUN npm install -g n
-RUN n 6.11.1
-RUN npm install -g bower
-RUN npm install -g grunt
-
-# Build lisk node binary
-RUN git clone https://github.com/LiskHQ/lisk-node.git
-WORKDIR lisk-node
-RUN ./configure
-RUN make
-RUN cp ./out/Release/node /tmp/lisk-node
-WORKDIR ../
-RUN sudo rm -rf lisk-node
-
-# Install PostgreSQL
-ADD scripts/setup_postgresql.Linux ./
-RUN bash ./setup_postgresql.Linux
+RUN apt-get install -y autoconf automake build-essential curl git gzip libtool nano python wget tar jq sudo postgresql-client-9.4
```

Comment:
Better to use the same Postgresql supported version in core.

Reply:
Core will be using postgres 9.6.6 in the next release so we should look to support that directly in this docker image.

Reply:
The latest version available on dockerhub is 9.6.5

As soon as 9.6.6 is available I will create another PR to get it in sync.